README.md

```diff
@@ -141,3 +141,7 @@ Inspired by [js-must-watch](https://github.com/bolshchikov/js-must-watch). Creat
     * [PyCon presentation](https://us.pycon.org/2015/schedule/presentation/362/)
     * Video: [youtube](https://www.youtube.com/watch?v=_AEJHKGk9ns) [00:25:20]/[pyvideo](http://pyvideo.org/video/3466/facts-and-myths-about-python-names-and-values)
     * [Slides and new article](http://nedbatchelder.com/text/names1.html)/[Original article](http://nedbatchelder.com/text/names.html)
+
+##2019
+
+https://courses.edx.org/courses/course-v1:MITx+6.00.1x+2T2019a/c
```

Comment:
Not found

Reply:
Hi . thanks for pointing out. have to remove the "C" at the end thats it.

Reply:
Done.